### ISSUE TITLE
ignore Twitter

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#236]
First with the Gittip Crisis, and then in removing 8chan and weev, we've attracted detractors. Haters, if you will. We are now a target of strongly felt ire and outrage, the expression of which, Twitter is optimized for. As a result, Twitter has become a huge emotional drain for me personally over the past year, and I've been experimenting lately with simply ignoring it, not only for me personally but also for Gratipay as a company. Following on from @tshepang's question at https://github.com/gratipay/gratipay.com/pull/3097#issuecomment-108511096, here's why I think this is in fact the right move for Gratipay as a company.

#### Sandman's Model

I've found Peter Sandman's [risk communication model](http://psandman.com/index-intro.htm) to be helpful in thinking about our situation. He discusses risk communication in terms of hazard vs. outrage. Hazard is how bad something is, and outrage is how mad people are about it. His main thesis is that the two are uncorrelated, and he recommends different strategies for different combinations of hazard and outrage:

| hazard | outrage | task | description |
|:----------|:-----------|:------|:---------------|
| high     | low        | precaution advocacy | alerting insufficiently upset people to serious risks. “Watch out!” |
| low      | high       | outrage management | reassuring excessively upset people about small risks. “Calm down.” |
| high     | high       | crisis communication | helping appropriately upset people cope with serious risks. “We’ll get through this together.” |
| intermediate | intermediate | sweet spot | dialoguing with interested people about a significant but not urgent risk. “And what do you think?” |

![colhandout](https://cloud.githubusercontent.com/assets/134455/7968877/5e993608-0a01-11e5-8804-80b6e807204f.gif)

#### Audience

I *love* Sandman's "sweet spot." I want to live there forever. And, for the first year or two of Gratipay's existence, our Twitter experience *was* sweet! We were indeed, "dialoguing with interested people," about interesting challenges. Yet as we grew and began to distinguish ourselves from other public actors, Twitter soured considerably, for two reasons. On the one hand, Twitter is optimized for the expression of outrage. On the other, our individuation has left a number of folks disappointed to the point of outrage that we did not turn out the way they wanted us to. We now have a number of people motivated to express outrage about Gratipay, and Twitter is a great place for them to do that.

Here's the catch: these people are not our audience. Our audience are the thousands of people who *are* on board with Gratipay's [mission](http://inside.gratipay.com/big-picture/mission) and [values](http://inside.gratipay.com/big-picture/brand/) as we're concretely pursuing them. *They're* the ones to whom we want to dedicate our precious fund of attention.

Sandman's model doesn't address this question of audience. We only have to engage in "outrage management" if we care to calm down the people who are upset. We can also choose to ignore the people who are outraged, and that's what we've been doing lately. It's nearly a self-referential point: the people who are prone to outrage over Gratipay's decisions are not Gratipay's audience.

#### Medium, Message

Twitter, as a technology, doesn't offer nearly the [emotional bandwidth](http://www.kapor.com/writing/emotional-bandwidth/) that's needed to maintain Sandman's sweet spot. As a result, I believe we should abandon it. Any Gratipay user that has a problem with Gratipay can contact us privately [via email](mailto:support@gratipay.com), or publicly [on GitHub](https://github.com/gratipay/gratipay.com/issues/new). In fact, this is what they&mdash;you!&mdash;have done! Anecdotally, my experience is that positive engagement with Gratipay is at a two-year high this past month, as we've been navigating the Gratipocalypse and the transition to Gratipay 2.0. We've got interesting challenges, and interested people working together to address them. This is the sweet spot. This is where we want to be.

I propose that we adopt this policy of ignoring Twitter for the foreseeable future. Of course, any one of us as an individual can do whatever we want on Twitter; I'm talking here about the corporate Gratipay Twitter account. Until we implement [email updates](https://github.com/gratipay/gratipay.com/issues/2521), any users that want to keep up with high-level announcements can follow [our blog](https://medium.com/gratipay-blog) (and yes, Medium publishes [an RSS feed](https://medium.com/feed/gratipay-blog)). As an addendum: let's be clear that the audience for our blog is our users, and *not* the general public; the general public is not an audience of ours.

**Comment by whit537** [comment]
P.S. I've personally abandoned IRC as well, for similar reasons. I downgraded IRC to a "Moribund" [channel](http://inside.gratipay.com/appendices/channels).

I just have to say that I *love* working on Gratipay more now than any time in the past year. I love looking at GitHub and at Freshdesk with no social tab. We're getting stuff done! We're kicking butt! Is anyone else really missing IRC or Twitter?

**Comment by tshepang** [comment]
It's great to hear that you are enjoying working on the project this much, especially given the recent crisis.

I feels sad to abandon Twitter, but it's understandable, given the attacks I've seen you get. I'm sorry for the pain, and am grateful that you keep going.

**Comment by tshepang** [comment]
Can you please explain this for a bit:

>  Anecdotally, my experience is that positive engagement with Gratipay is at a two-year high this past month

**Comment by whit537** [comment]
> What do you think caused it?

I suspect that [notifying all users via email](https://github.com/gratipay/gratipay.com/issues/3385#issuecomment-100040443) helped, as well as [reaching out to specific users via email](https://github.com/gratipay/inside.gratipay.com/issues/198) to jump-start the reboot. After that I would say that making sure to keep up with email support and GitHub have been crucial: we learned [early on from FeeFighters](http://blog.gittip.com/post/25565694263/how-not-to-get-acquired/) that neglecting support during a crisis only deepens the crisis.

What do you think?

**Comment by tshepang** [comment]
I just didn't expect that you felt so positive. It just looked stressful to me.

**Comment by whit537** [comment]
Sure, it has been and continues to be a lot of work. But we're in the sweet spot, which means the work is ultimately satisfying, because we're accomplishing something awesome together as a group! :rocket: :full_moon_with_face: :dancer:

**Comment by techtonik** [comment]
@whit537 I'd be very interested in maintaining Twitter account for GP. It looks like a real challenge that needs manipulations from both dark side and light side of the world. But I am not sure that we are ready to deal with the dark side just yet. It is a full time activity and I am just not in a position (mostly financially) to get the GP out of this pit.

If we're going to abandon the Twitter, that should come with an explanation. About social justice, activism, conflicts. having fun on the nets and the isolated spiritual worlds we are living in that are connected by invasive global economy. I feel like there something huge hiding behind that, and if put right, it can move the talks into right direction of looking for the right cause of the problems, which are very deep and interesting.

**Comment by mattbk** [comment]
+1 to @techtonik.

Twitter is what you make of it, but unfortunately there are very few positive mentions of Gratipay in comparison to the negative ones by certain individuals.  That's a marketing problem, not a medium problem.

**Comment by whit537** [comment]
> If we're going to abandon the Twitter, that should come with an explanation.

@techtonik You think we should put https://github.com/gratipay/inside.gratipay.com/issues/236#issue-84743805 in a blog post?

> That's a marketing problem, not a medium problem.

@mattbk I think the fundamental question here is, who is our audience? And then, how are we going to communicate with them?

**Comment by dcht00** [comment]
@whit537 

TLDR:
* How should an open company handle public-facing communication?
* You'd burn out less if there'd be other people taking the heat. Where's the barrier?

Please point me to any resources on that if existing.

I have a similar situation with the Hackbase.

First the website:
scroll to bottom of totalism.org and you will see it's an editable pad. ("Wiki" would serve same purpose).
So far I never had (detected) vandalism (!? where are you vandals). I leave it up to the situation to see where that takes me.

For email, I want to simply point info@ email to a mailing list. Soon.

With social media, I was thinking of having a cyborg-hydra multi-person, namely, account that retweets everything that is tagged in a specific way.
An update to that (for bigger projects) would be having a mezzanine layer where the cyborg-hydra takes care internally to be somewhat consistent, possibly with a software that takes into account contributions of individuals forming it, a P2P reputation network, or so such. But nothing too fancy.

**Comment by whit537** [comment]
> With social media, I was thinking of having a cyborg-hydra multi-person, namely, account that retweets everything that is tagged in a specific way.

We tried this a bit with [OCI](https://twitter.com/OpenCompanyOrg). Bottom line is having the right people involved.

**Comment by whit537** [comment]
We're *de facto* doing this right now. Someone reticket if/when the time is right to reconsider.
